docs/node/RPC Category/16-loan.md

```diff
@@ -549,7 +549,7 @@ interface loan {
 
 interface TakeLoanMetadata {
   vaultId: string
-  amounts: string | string[] // amount@symbol
+  amounts: string // amount@symbol
```

Comment:
```suggestion
  amounts: string | string[] // amount@symbol
```
takeloan support multiple tokens at once

Reply:
HAHAHA TakeLoanMetadata not related this this PR right? Will fix anyway.

Reply:
yea, but this PR updates that structure. should be a mistake.